tests/functional/test_basic.py

```diff
@@ -81,30 +75,28 @@ def MyEXE(*args, **kwargs):
 
     pyi_builder.test_source("print('Hello Python!')")
 
+
 def test_base_modules_regex(pyi_builder):
     """
     Verify that the regex for excluding modules listed in
     PY3_BASE_MODULES does not exclude other modules.
     """
-    pyi_builder.test_source(
-        """
+    pyi_builder.test_source("""
```

Comment:
Why were opening """ been moved to line above? Couple of test cases below, the opposite rule is taken...